zeppelin-client/src/main/java/org/apache/zeppelin/client/ZeppelinClient.java

```diff
@@ -559,6 +559,24 @@ public String addParagraph(String noteId, String title, String text) throws Exce
     return jsonNode.getObject().getString("body");
   }
 
+  /**
+   * delete paragraph with noteId and paragraphID
+   *
+   * @param noteId
+   * @param paragraphId
+   * @throws Exception
```

Comment:
It would be nice if there were more helpful descriptions. Of course you've referenced the existing code, but isn't there a need to copy the existing bad parts?